### PR TITLE
Po 408 intial classes for file handling

### PR DESCRIPTION
**Pull request by DeclanClarkeCGI**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/opal/model/dto/StandardBankingFile.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            .build();
+    }
+
+    public static String toString(StandardBankingFile file) {
+        String spaces = " ".repeat(18);
+        StringBuilder sb = new StringBuilder();
+        sb.append(file.getHeader());
+        for (FinancialTransaction financialTransaction : file.getFinancialTransactions()) {
+            sb.append(System.lineSeparator())
+                .append(financialTransaction.getBranchSortCode())
```
Comment (RustyHMCTS) [inline src/main/java/uk/gov/hmcts/reform/opal/model/dto/StandardBankingFile.java:84]: Is this property and all the following guaranteed to be of the correct length? i.e. could they be shorter/longer than that required to correctly construct a banking file?

File: src/main/java/uk/gov/hmcts/reform/opal/repository/AntCtAmalgamatedRepo.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.opal.repository;
+
+//import org.springframework.data.jpa.repository.JpaRepository;
```
Comment (RustyHMCTS) [inline src/main/java/uk/gov/hmcts/reform/opal/repository/AntCtAmalgamatedRepo.java:3]: Remove comments?

**Review by RustyHMCTS — APPROVED** [review]

**Review by TomReed — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/opal/model/dto/StandardBankingFile.java (hunk 1 above)
Comment (DeclanClarkeCGI) [reply src/main/java/uk/gov/hmcts/reform/opal/model/dto/StandardBankingFile.java:84]: Documentation shows these fields should be consistent in placement, the files themselves have no separation value so we have to use the expected positions

File: src/main/java/uk/gov/hmcts/reform/opal/repository/AntCtAmalgamatedRepo.java (hunk 2 above)
Comment (DeclanClarkeCGI) [reply src/main/java/uk/gov/hmcts/reform/opal/repository/AntCtAmalgamatedRepo.java:3]: Will be removed when these classes are updated for data access.

**Pull request merged**